client/src/ui/atoms/search/index.tsx

```diff
@@ -22,16 +22,11 @@ type SearchProps = {
 
 export const Search = ({
   name,
-  extraClasses,
-  id,
-  isDisabled,
```

Comment:
If you remove them here (because they're presumably no longer used), shouldn't we also remove them in the `SearchProps` above (lines 5 f.)?

Reply:
I thought to do that, but then I thought perhaps there is a valid use case for these and keeping them around does not cause any problem. Either works for me really.

Reply:
My best guess is that when passing these formerly unused and now removed props (like `extraClasses`) into this `Search` component, they will just get ignored silently, rather than being applied to the first child. As this is a potential cause of issue, I would opt for removing them from the `SearchProps` as well, especially since the only props really used are `name`, `placeholder`, `onBlurHandler` and `onChangeHandler`:

https://github.com/mdn/yari/blob/81d6989fcb74399ee9df3facae5e91bb537ca719/client/src/plus/search-filter/index.tsx#L80-L85

I know that Vue _does_ pass unknown props as attributes to the first child, but my impression is that React doesn't do magic stuff like this.

Reply:
Works for me. Will update.